src/jit/flowgraph.cpp

```diff
@@ -9021,6 +9021,49 @@ void Compiler::fgSimpleLowering()
                     // Add in a call to an error routine.
                     fgSetRngChkTarget(tree, false);
                 }
+
+#if FEATURE_FIXED_OUT_ARGS
+                if (tree->gtOper == GT_CALL)
+                {
```

Comment:
I though about that worried that calls could be created in `fgSetRngChkTarget` -- though it looks like it doesn't actually add calls despite the comment.